addons/trivy-operator/metadata.yaml

```diff
@@ -0,0 +1,17 @@
+name: trivy-operator
+version: 2.3.2
+description: Kubernetes Operator based on the open-source container vulnerability scanner Trivy
+icon: https://github.com/devopstales/helm-charts/raw/main/icons/trivy.png
+url: https://github.com/devopstales/trivy-operator
```

Comment:
This image URL maybe cannot show correctly. Please use `https://raw.githubusercontent.com/devopstales/helm-charts/main/icons/trivy.png`